docs/src/main/tut/docs/faq.md

```diff
@@ -96,3 +97,57 @@ But, of course, the values we need are still there:
 secret.password.value 
 secret.apiKey.value
 ``` 
+
+### How do I debug "implicit not found" errors?
```

Comment:
I think this should be in its own section because many developers could be interested

Reply:
I guess so. But even though it's technically important, I can't seem to like it as a top-level section on our documentation, on par with things like "Supporting New Types" and "Error Handling". The problem is not even specific to PureConfig...

I can see two alternatives here:

- A "Troubleshooting" top-level section that has not only this, but also common problems our users have (like mixing up naming conventions). If that if the section is called "Troubleshooting" users having problems with implicits not found will go there to check;
- Add all FAQ questions to the side bar, giving users direct links to the common questions they are having. Similarly, by seeing a related question at the side bar users will find it easily. 

What do you think?